### ISSUE TITLE
How to enable HTTP to HTTPS redirect?

### ISSUE THREAD

**Issue by peterver** [digitalocean/digitalocean-cloud-controller-manager#96]
I've been looking through the provided annotations and I can't seem to find one that toggles the 'Http to Https' redirect option on DO load balancers?

( I went through this file : https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/master/do/loadbalancers.go )

At the moment my setup is like this ( terraform based ) : 

```
...
resource "kubernetes_service" "srv" {
        metadata {
                name = "..."
                namespace = "${var.m_namespace}"
                annotations {
                        "service.beta.kubernetes.io/do-loadbalancer-protocol" = "http"
                        "service.beta.kubernetes.io/do-loadbalancer-algorithm" = "round_robin"
                        "service.beta.kubernetes.io/do-loadbalancer-tls-ports" = "443"
                        "service.beta.kubernetes.io/do-loadbalancer-certificate-id" = "${var.m_certificate_id}"
                }
        }

        spec {
                selector {
                        app = "..."
                }
                port = [
                        {
                                name = "http"
                                port = 80
                                protocol = "TCP"
                                target_port = 8080
                        }, {
                                name = "https"
                                port = 443
                                protocol = "TCP"
                                target_port =  8080
                        }
                ]

                type = "LoadBalancer"
        }
}
...
```

What would be the expected annotatation for this ( if it is available ) ? Or what would be an alternative ? For now I'm making the adjustment manually once the load balancer has been provisioned, but that's really not the expected way of working for me :/ 

Loving the module btw guys :D good job !

**Comment by peterver** [comment]
In case this was not added yet I can run an extra terraform resource that takes the loadbalancer created by cloud control manager through k8s and sets the boolean flag automatically ( but if possible I wouldn't want to do that sicne it would be multi-step rather than a single step :/ 

https://www.terraform.io/docs/providers/do/r/loadbalancer.html#redirect_http_to_https

**Comment by peterver** [comment]
I've made a pull request that adds this functionality #98

**Comment by peterver** [comment]
@andrewsykim once released this issue can be closed ^_^ ! Thanks for the help :)

**Comment by andrewsykim** [comment]
Fixed in v0.1.6, thanks for the patch @peterver
